app/components/@settings/core/types.ts

```diff
@@ -28,7 +28,7 @@ export const TAB_LABELS: Record<TabType, string> = {
   data: 'Data Sources',
   github: 'GitHub',
   'deployed-apps': 'Deployed Apps',
-  members: 'Members',
+  members: 'Users',
```

Comment:
for copy - let's keep the term members. it's more natural for organization to use the term member vs user.